### ISSUE TITLE
About Mysql8.0.12

### ISSUE THREAD

**Issue by wooir** [phpmyadmin/phpmyadmin#14623]
### Additional context

- phpMyAdmin version:4.8.3

Does phpmyadmin4.8.3 support mysql8.0.12? If not, will the next version support it?

**Comment by williamdes** [comment]
@wooir Can you please explain what you call "support" ?

**Comment by ibennetch** [comment]
In general, most things should work however due to MySQL making many breaking changes in this minor point release, we have several known issues, including:

* https://github.com/phpmyadmin/phpmyadmin/issues/14245
* https://github.com/phpmyadmin/phpmyadmin/issues/14220
* https://github.com/phpmyadmin/phpmyadmin/issues/14217

Personally, I’m pretty irritated at MySQL for making such major changes without a more drastic change of the version or working with developers to ensure tools that use the database are updated (by which I mean that the PHP bug report about the new authentication is five months old with 111 votes and no fix).

Anyway, to answer your question it is not fully compatible at this time but you should be able to work around most issues. There’s no time table for when these things will be fixed (in particular, right now we’re stuck waiting for PHP themselves to become compatible).

**Comment by wooir** [comment]
@williamdes My English is not very good. In Chinese, "support" can be expressed as "compatibility" in some cases.

**Comment by wooir** [comment]
@ibennetch So far, I am only dissatisfied with #14217 , although I can do database creation and password settings after creating a user. Are you saying that this problem will not be corrected for the time being?

**Comment by OlafvdSpek** [comment]
@ibennetch 
> Personally, I’m pretty irritated at MySQL for making such major changes without a more drastic change of the version
8.0 is a major new version isn't it?

Isn't that issue a PHP issue?

**Comment by ibennetch** [comment]
@wooir that bug is something I’d like to have fixed but haven’t had time to work on myself. I can’t say when it will be fixed, but it’s pretty high on the list of bugs I’d like to fix. 

The authentication bug is dependent on PHP fixing their plugin so I can’t even give a estimate for that one.

**Comment by ibennetch** [comment]
Olaf, I’m away from my notes on the matter at the moment, but by memory the breaking changes happened between MySQL 8.0.4 and 8.0.11. I remember testing the release candidates of 8.0.0 pretty extensively and (again, this is from memory) I don’t think they mentioned the plan to deprecate and remove the features are using until 8.0.11 was released.

In my opinion, changing the authentication architecture and methods of user management from version 8.0.4 to 8.0.11 violates semantic versioning principals.

Again, the details of that are from memory so I might have gotten something wrong but I do remember that I didn’t see anything documented prior to the changes being released.

**Comment by OlafvdSpek** [comment]
You're right, I didn't realize 8.0.3 and 8.0.4 were release candidates already and 8.0.5 - 8.0.10 were skipped.
